db/models/Review.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (sequelize, Sequelize) => {
+  const Review = sequelize.define('Review', {
```

Comment:
I'm not sure where the review scores are stored. That may be worth naming differently or making it it's own field.